test/models/department_test.rb

```diff
@@ -2,12 +2,13 @@
 #
 # Table name: departments
 #
-#  id          :integer          not null, primary key
-#  name_dw     :string           not null
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
-#  code_dw     :string           default(""), not null
-#  name_dspace :string
+#  id                   :integer          not null, primary key
+#  name_dw              :string           not null
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#  code_dw              :string           default(""), not null
+#  name_dspace          :string
+#  authority_key_dspace :string
```

Comment:
To match the rest of this test suite, I think a `valid without authority_key_dspace` test would be good.

Reply:
Yeah, good call - I just pushed a commit to add this test.